Python/optimizer.c

```diff
@@ -1659,4 +1668,61 @@ _Py_Executors_InvalidateAll(PyInterpreterState *interp, int is_invalidation)
     }
 }
 
+void _Py_Executor_Invalidate(_PyExecutorObject *executor)
+{
+   if (executor->vm_data.valid) {
+    unlink_executor(executor);
+    executor_clear(executor);
+   }
+}
+
+void
+_Py_Executors_InvalidateOld(PyInterpreterState *interp, int is_invalidation)
```

Comment:
Perhaps this should be renamed to `InvalidateCold`, given the direction this ended up taking? Happy to update/take suggestions

Reply:
You read my mind! Yeah, let's rename this.

Reply:
Also, I don't think we need the `is_invalidation` arg here. It's always called with `0`.